firestore-bigquery-export/functions/src/index.ts

```diff
@@ -70,3 +78,60 @@ exports.fsexportbigquery = functions.firestore
       logs.error(err);
     }
   });
+
+exports.fsimportexistingdocs = functions.tasks
+  .taskQueue()
+  .onDispatch(async (data) => {
+    const runtime = getExtensions().runtime();
+    if (!process.env.DO_BACKFILL) {
+      await runtime.setProcessingState(
+        "PROCESSING_COMPLETE",
+        "No existing documents imported into BigQuery."
```

Comment:
No backfill requested. No existing ... into BigQuery. If you change your mind and want to import them, reconfigure your extension and choose 'yes' for "Import existing Firestore documents into BigQuery?".
Also: Maybe get Kevin to look at this wording. He's better at this stuff than I am :)